master/buildbot/interfaces.py

```diff
@@ -129,6 +129,236 @@ def getAddress(user):
         be reached by email), or a Deferred which will fire with the same."""
 
 
+class ITestResult(Interface):
```

Comment:
I think we should kill this interface and remove its use from subunit.  We're not storing test results in the data API (and I don't think anything actually uses them).  From what I can tell this commit is just rearranging `interfaces.py`, in which case it's fine to just leave it as-is.